services/navigation/builtin/builtin_test.go

```diff
@@ -425,34 +524,52 @@ func TestStartWaypoint(t *testing.T) {
 			test.That(t, <-statusChannel, test.ShouldEqual, arrivedAtWaypointMsg)
 			currentInputsShouldEqual(ctx, t, kinematicBase, pt3)
 		})
-		t.Run("Calling SetMode cancels current and future MoveOnGlobe calls", func(t *testing.T) {
-			// Set manual mode to ensure waypoint loop from prior test exits
-			err = deleteAllWaypoints(ctx, ns)
-			test.That(t, err, test.ShouldBeNil)
-			for _, pt := range points {
-				err = ns.AddWaypoint(ctx, pt, nil)
+
+		// Calling SetMode cancels current and future MoveOnGlobe calls
+		cases := []struct {
+			description string
+			mode        navigation.Mode
+		}{
+			{
+				description: "Calling SetMode manual cancels current and future MoveOnGlobe calls",
+				mode:        navigation.ModeManual,
+			},
+			{
+				description: "Calling SetMode explore cancels current and future MoveOnGlobe calls",
+				mode:        navigation.ModeExplore,
+			},
+		}
+
+		for _, tt := range cases {
+			t.Run(tt.description, func(t *testing.T) {
+				// Set manual mode to ensure waypoint loop from prior test exits
+				err = deleteAllWaypoints(ctx, ns)
 				test.That(t, err, test.ShouldBeNil)
-			}
+				for _, pt := range points {
+					err = ns.AddWaypoint(ctx, pt, nil)
+					test.That(t, err, test.ShouldBeNil)
+				}
 
-			// start navigation - set ModeManual first to ensure navigation starts up
-			err = ns.SetMode(ctx, navigation.ModeWaypoint, map[string]interface{}{"experimental": true})
+				// start navigation - set ModeManual first to ensure navigation starts up
+				err = ns.SetMode(ctx, navigation.ModeWaypoint, map[string]interface{}{"experimental": true})
```

Comment:
Do we still need 'experimental' here? I think this was used for feature flagging GPS navigation.

Reply:
Removing it caused no issues, will confirm with motion team

Reply:
Confirmed with team, this has been deprecated for the last month